### PR TITLE
update services moved onto hardcase and removed services

### PR DESCRIPTION
**Pull request by theycallmemac**
Updating some information on the follow:

- services/git
- services/paste

Removing information on:

- web/pubcookie
- web/apachemodules

### CODE REVIEW

File: docs/services/git.md [hunk 1]
```diff
@@ -20,9 +20,8 @@ Permissions on the gitea_token.secret must be 740 and owned by git:gitea
 ## Redbrick Special Notes
 
 - The giteadmin credentials are in the passwordsafe
-- SSH is set to `10022` because otherwise it would clash with the host's real SSH
-- OpenID and user registration are disabled to stop non-RB users joining,
-only LDAP works.
+- SSH is set to `10022` because otherwise it would clash with the host's real SSH.
```
Comment (butlerx) [inline docs/services/git.md]: This was actually disabled in the migrations to nix.

File: docs/services/paste.md [hunk 2]
```diff
@@ -1,18 +1,11 @@
 # Pastebin
 
-Redbrick uses [hastebin](https://github.com/seejohnrun/haste-server) for pastes
+Redbrick currently uses m1cr0man's fork of [Privatebin](https://github.com/m1cr0man/PrivateBin) as a paste utility.
```
Comment (butlerx) [inline docs/services/paste.md]: Did we not just switch back from the fork?

File: docs/services/git.md (hunk 1 above)
Comment (theycallmemac) [reply docs/services/git.md]: I'll remove the SSH point then, thanks.

File: docs/services/paste.md (hunk 2 above)
Comment (theycallmemac) [reply docs/services/paste.md]: https://github.com/redbrick/nix-configs/blob/master/packages/privatebin/default.nix

Still uses the fork.

Comment (butlerx) [reply docs/services/paste.md]: cool

Comment (butlerx) [reply docs/services/paste.md]: https://github.com/redbrick/nix-configs/pull/39 will switch back to upstream

**Comment by theycallmemac** [comment]
CI failing due to an exception cropping up during the pip install phase

File: docs/services/paste.md (hunk 2 above)
Comment (m1cr0man) [reply docs/services/paste.md]: Yep this is merged now

**Review by m1cr0man — CHANGES_REQUESTED** [review]
Looks good juts the bit about the privatebin fork needs to change

**Review by m1cr0man — DISMISSED** [review]

**Review by butlerx — DISMISSED** [review]

**Review by butlerx — APPROVED** [review]

**Review by m1cr0man — APPROVED** [review]

**Pull request merged**